specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/examples/Subscription_TestQuery.json

```diff
@@ -0,0 +1,71 @@
+{
+  "parameters": {
+    "api-version": "2017-04-01-preview",
+    "subscriptionId": "56b5e0a9-b645-407d-99b0-c64f86013e3d",
+    "location": "West US",
+    "streamingJob": {
+      "properties": {
+        "sku": {
+          "name": "Standard"
+        },
+        "eventsOutOfOrderPolicy": "Drop",
+        "outputErrorPolicy": "Drop",
+        "eventsOutOfOrderMaxDelayInSeconds": 0,
+        "eventsLateArrivalMaxDelayInSeconds": 5,
+        "dataLocale": "en-US",
+        "compatibilityLevel": "1.0",
+        "inputs": [
+          {
+            "properties": {
+              "type": "Stream",
+              "datasource": {
+                "type": "Raw",
+                "properties": {
+                  "payload": "{}",
```

Comment:
NIT: Only one of payload or payloadUri can specified, but not both. Might be good to also put that in the properties description.